src/mouse/default_gutter_handler.js

```diff
@@ -94,7 +119,7 @@ function GutterHandler(mouseHandler) {
         if (mouseHandler.$tooltipFollowsMouse) {
             moveTooltip(mouseEvent);
         } else {
-            var gutterElement = mouseEvent.domEvent.target;
+            var gutterElement = gutter.$lines.cells[row].element.children[1];
```

Comment:
what is `gutter.$lines.cells[row].element.children[1]`? is there a better way to get that element instead of relative positioning of element among other children?

Reply:
this would get the same result:
```
var gutterElement = gutter.$lines.cells[row].element.querySelector("[class*=ace_icon]");
```